concepts/05 UI Components/Scheduler/00 Getting Started with Scheduler/15 Edit Appointments.md

```diff
@@ -0,0 +1,88 @@
+Users can manage appointments in the following ways:
+
+* Add new appointments
+
+* Update appointments in an edit form
+
+* Resize an appointment to change its duration
+
+* Drag and drop an appointment to reschedule it
+
+* Remove appointments
+
+Edit operations are enabled by default. To disable an individual operation, assign **false** to the corresponding property in the [editing](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/editing/) object. In this tutorial, we use the [allowDragging](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/editing/#allowDragging) property to disable appointment drag-and-drop.
```

Comment:
1) `Edit operations are enabled by default.`
We need to rewrite this without using "by default". For example, can we say the following?

`Edit operations are enabled (the default setting).`

Or do you mean:

`Edit operations are initially enabled.`

2) `To disable an individual operation, assign **false** to the corresponding property in the [editing](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/editing/) object.`
Can we change this to the following?

`To disable an individual operation, set the corresponding property in the [editing](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/editing/) object to **false**.`